.changeset/little-coins-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": patch
+---
+
+Enable "Credit Card" option in Buy screen in ConnectButton
```

Comment:
requires more of a changeset, what countries does it work in, other restrictions etc